WordPress/RN/Gutenberg/GutenbergController.swift

```diff
@@ -141,6 +219,7 @@ extension GutenbergController: PostEditorNavigationBarManagerDelegate {
     }
 
     func navigationBarManager(_ manager: PostEditorNavigationBarManager, publishButtonWasPressed sender: UIButton) {
+        requestHTMLReason = .publish
```

Comment:
agreed. it is absolutely not an ideal solution, better solution would be to pass the bridge the reason of html request and take back that reason in the callback. I did this only because the bridge operation only lasted 0.004 seconds, users can't tap another place in this much little time so in terms of practical world I can say that we are safe, at least for now.
but still let's discuss on this and try to find a better solution.

Reply:
> the bridge operation only lasted 0.004 seconds

That is reassuring :)